Dockerfile

```diff
@@ -1,18 +1,157 @@
-FROM ubuntu:bionic
-MAINTAINER Fmstrat <fmstrat@NOSPAM.NO>
+#
+# To build the RPMs
+#
+FROM rockylinux:8.5 as src
```

Comment:
eventually it should be FROM ark_case I believe... as I understand it, all the other images should come from the same base image.

Reply:
Agreed ... I just didn't have the image name handy :)

Reply:
Hmm... @naveen-armedia's ark_solr Dockerfile, which he just updated, has `FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest`